src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/pooling.hpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <tuple>
+#include <string>
+#include <vector>
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "common_test_utils/test_enums.hpp"
+
+namespace ov {
+namespace test {
+typedef std::tuple<
+        ov::test::utils::PoolingTypes,  // Pooling type, max or avg
```

Comment:
Do we have the same enum in ov? if yes, please replace by OV

Reply:
What you mean? It is already in ov namespace (common_test_utils/test_enums.hpp)

Reply:
I mean not test namespace. I think we should have same types in OV lib

Reply:
I checked, there are no PoolingTypes in main lib now.